src/service/command-executor.js

```diff
@@ -94,9 +98,32 @@ class KatalonCommandExecutor extends BaseKatalonCommandExecutor {
     super(info);
     this.teamId = info.teamId;
     this.projectId = info.projectId;
+    this.extraFiles = info.extraFiles;
+  }
+
+  async downloadExtraFiles(extraFiles, ksProjectPath, jLogger) {
+    const extraFileDownloads = [];
+    let shouldDownloadExtraFile = false;
```

Comment:
explain shouldDownloadExtraFile variable

Reply:
I update new logic so this variable is removed